atlas-content-modeler/tutorial/models/taxonomies/query.mdx

```diff
@@ -0,0 +1,117 @@
+---
+slug: /atlas-content-modeler/tutorial/models/taxonomies/query
+title: Atlas Content Modeler Taxonomies - Query
+description: Learn how to query Models by Taxonomy in Atlas Content Modeler.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../../introduction.mdx).
+
+## Query WPGraphQL for a Taxonomy
+
+Learn how to query taxonomies using WPGraphQL.
```

Comment:
```suggestion
Learn how to query models by taxonomy using WPGraphQL.
```
Missed this before, but I think it should read same/similar to the description above so that it clarifies you're querying/filtering content by taxonomy.